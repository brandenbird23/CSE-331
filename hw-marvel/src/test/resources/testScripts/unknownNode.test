# Tests finding a path to a node that doesn't exist in the graph

LoadGraph nfl nflPlayers.csv

FindPath nfl GENO-SMITH DAVE-KRAIG
FindPath nfl DAVE-KRAIG RUSSEL-WILSON
FindPath nfl JOE-BURROW JOSH-ALLEN