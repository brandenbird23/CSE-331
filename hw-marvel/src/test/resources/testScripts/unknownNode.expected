# Tests finding a path to a node that doesn't exist in the graph

loaded graph nfl

unknown: DAVE-KRAIG
unknown: DAVE-KRAIG
unknown: JOE-BURROW
unknown: JOSH-ALLEN